definitions/divorce/json/CaseEventToFields/CaseEventToFields-bailiff-nonprod.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "LiveFrom": "81/02/2021",
```

Comment:
wow :)

Reply:
Please change it. Even though CCD ignores it maybe they internally cast it to `Date` and it may throw exception (now or in the future if they start doing it :) )